cfme/tests/v2v/test_rest_migrations.py

```diff
@@ -189,3 +191,39 @@ def _cleanup():
     results = appliance.rest_api.response.json()["results"]
     assert results[0]["success"] is False
     assert results[1]["success"] is True
+
+
+@pytest.mark.parametrize("transformation_method", ["SSH", "VDDK"], ids=["SSH", "VDDK"])
```

Comment:
You don't really need to use `ids` here since parameter names and ID are both same.